src/globalFilters/FilterField.vue

```diff
@@ -10,8 +10,11 @@
             name="setSelectionThreshold"
             @change="updateFilterValueFromDropdown($event, filter.comparator, $event.target.value)"
           >
-            <option key="NONE" value="NONE">
-              None
+           <option value="NONE" v-if="filter.defaultLabel">
```

Comment:
can we make this a computed value `defaultLabel` then we can get rid of `v-if` and it'll just cache the value since it won't change.

Reply:
Sure - in general is the preference to keep logic out of vue?

Reply:
you could even just set it on mounted, if there are not other computed values. Since this will not change that may make sense

Reply:
Removed the v-if and this seems to be the cleanest way to compute it: 

```
           <option value="NONE">
            {{ filter.defaultLabel || "None" }}
            </option>
```